cinema/serializers.py

```diff
@@ -69,6 +95,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
         source="cinema_hall.capacity", read_only=True
     )
     tickets_available = serializers.IntegerField(read_only=True)
+    movie_image = serializers.ImageField(source="movie.image", read_only=True)
```

Comment:
also you don't need to duplicate movie_image here, if you defined it in the MovieSessionSerializer